openquake/calculators/event_based_risk.py

```diff
@@ -360,27 +371,23 @@ def build_loss_maps(self):
 
     # ################### methods to compute statistics  #################### #
 
-    def build_stats(self, builder):
-        """
-        Compute all statistics for all assets starting from the
-        stored loss curves. Yield a statistical output object for each
-        loss type.
-        """
+    def _collect_data(self):
```

Comment:
This might be renamed into `_collect_all_data`, to make it more explicit that we also have `_collect_specific_data`